pandas/tests/extension/test_floating.py

```diff
@@ -25,12 +25,17 @@
 from pandas.tests.extension import base
 
 
-def make_data():
+def make_data(with_nas: bool = True):
+    na_value = pd.NA if with_nas else np.nan
+    # `not with_nas` -> case without pd.NA that can be cast to
+    #  floating ndarray losslessly
+    # TODO: get cases with np.nan instead of pd.NA GH#39039
```

Comment:
I don't really understand why this is needed? What does it test?

Reply:
Until #39039 is addressed so it is possible to make entries with np.nan instead of pd.NA, nothing.  Once that is addressed, this creates cases where it is possible to set a FloatingArray into a Series/column backed by ndarray[float64]

Reply:
Then I would leave that out until that is addressed. Because now this only doubles the test run for `test_floating.py` for nothing.